cryptoki-rustcrypto/src/rng.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+use signature::rand_core::{CryptoRng, Error as RndError, RngCore};
+use thiserror::Error;
+
+use crate::SessionLike;
+
+#[derive(Debug, Error)]
+pub enum Error {}
+
+/// [`Rng`] is a PKCS#11-backed CSPRNG.
+pub struct Rng<S: SessionLike>(S);
+
+// TODO(baloo): check for CKF_RNG bit flag (CK_TOKEN_INFO struct -> flags)
```

Comment:
Was this resolved, or will it result in a new issue once this is merged?

Reply:
No, I'll need to make an API change to session to get back the `Pkcs11` object and get_token_info on it.